daffodil-lib/src/main/scala/org/apache/daffodil/util/DefaultValidator.scala

```diff
@@ -100,3 +111,35 @@ object Validator extends NoBindingFactoryAdapter {
   }
 }
 
+class DefaultValidatorFactory extends ValidatorFactory {
+  def name(): String = "default" /// todo;; or should it be "xerces"
+
+  // to not muck around too much with the original logic above just wrapping access to the
```

Comment:
I dont really want to touch the original Validator implementation and the caching approach it uses.  So this adapter class wraps it.  It also implements the Xerces specific SAX error handling that the ParseResult previously provided.

I think its also a good example of how you can use a singleton objects with inexpensive wrappers through the factory interface (this is a little complex for an example, but the concept is there).

Reply:
This prevents the default validator from being a good motivation for the factory API pattern here. Instead you end up wondering why the factory idiom is here at all if the factory is just going to do nothing. 

I think the xerces default validator - all the cache complexity goes away if you actually implement the factory pattern.

Reply:
A rename for clarity might be useful too,  `XercesValidator`

Reply:
Refactored the default validator to use the factory pattern.